src/index.js

```diff
@@ -1,19 +1,19 @@
 // @flow
-'use strict';
```

Comment:
I'd actually like to leave this, because we don't have any babel transform adding it for us.